README.md

```diff
@@ -95,6 +95,55 @@ However, here are some rules that we'd like to emphasize since they are not visi
 
 For project ideas relating to the Typelevel ecosystem in the categories of AI/ML, serverless, data streaming, observability, and systems programming, please visit https://typelevel.org/gsoc/ideas/.
 
+### Scala Native / Scala.js Projects
+
+| Title                   | Scala Bazel Rules for Scala.js and Scala Native |
```

Comment:
This one is a bit problematic because we're not a maintainers of bazelbuild/rules_scala 

@liucijus, @simuons as a code owners would you be interested in allowing for this changes to be merged into main repository? The alternative approach would be have an alternative repository for rules dedicated only for Scala Multiplatform 

Also I'm look for a co-mentor, anyone interested in? @mbland you'd to willing to help as an active contributor?

Reply:
I'm honored by the offer, but I must decline. However, I'm happy to provide input if the need arises, particularly as it pertains to the Bazel side of things.